packages/shared/src/components/profile/UpvoterListPlaceholder.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactElement, memo } from 'react';
+import { PlaceholderList } from '../cards/PlaceholderList';
+
+export interface UpvoterListPlaceholderProps {
+  placeholderAmount?: number;
+}
+
+export function UpvoterListPlaceholder({
+  placeholderAmount = 5,
+}: UpvoterListPlaceholderProps): ReactElement {
+  return (
+    <div className="flex flex-col">
+      {Array(placeholderAmount)
+        .fill(0)
+        .map(() => (
+          <PlaceholderList key={Math.random()} />
+        ))}
+    </div>
+  );
+}
+
+export default memo(UpvoterListPlaceholder);
```

Comment:
Why do you memoize this component?

Reply:
I'm probably over-paranoid about the loop every re-render. But after thinking about it now, this might not actually be needed since this will be destroyed once the actual values are generated.